src/ui/components/NetworkMonitor/useNetworkContextMenu.tsx

```diff
@@ -22,6 +22,10 @@ export default function useNetworkContextMenu(row: Row<RequestSummary>) {
     dispatch(setFocusRegionBeginTime(beginTime!, true));
   };
 
+  const copyAsCurl = () => {
+    navigator.clipboard.writeText(`curl '${row.values.url}'`);
```

Comment:
I think we also want headers copied as well. Basically we want to be 1.1 with what chrome devtools does